reStream.sh

```diff
@@ -16,11 +17,24 @@ loop_wait="true"
 loglevel="info"
 
 # check if we are able to reach the remarkable
-if ! ssh "$ssh_host" true; then
-    echo "$ssh_host unreachable"
+if ! ssh -o ConnectTimeout=1 "$ssh_host_usb" true; then
+  if ! [ -z "$ssh_host_wifi" ]; then
+    if ! ssh "$ssh_host_wifi" true; then
+      echo "$ssh_host_wifi unreachable"
+      exit 1
```

Comment:
exit if none is reachable

Reply:
Maybe use `[ -n ... ]` instead of `!  [-z ...]`

Reply:
fixed in new commit